freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -581,12 +553,12 @@ public Host showHost(String fqdn) throws FreeIpaClientException {
             return response;
         } catch (Exception e) {
             String message = String.format("Invoke FreeIpa failed: %s", e.getLocalizedMessage());
-            LOGGER.error(message, e);
+            LOGGER.warn(message);
             OptionalInt responseCode = extractResponseCode(e);
             throw FreeIpaClientExceptionUtil.convertToRetryableIfNeeded(new FreeIpaClientException(message, e, responseCode));
         } catch (Throwable throwable) {
             String message = String.format("Invoke FreeIpa failed: %s", throwable.getLocalizedMessage());
-            LOGGER.error(message, throwable);
+            LOGGER.warn(message);
```

Comment:
why not to log and why not error?

Reply:
the throwable is set as the cause of the FreeIpaClientException and then rethrown. I feel that the caller should be responsible for logging/handling the exception. Seeing an error stacktrace in the logs at this point is misleading.